integration-cli/docker_utils.go

```diff
@@ -338,11 +339,14 @@ func sockRequest(method, endpoint string, data interface{}) (int, []byte, error)
 
 	res, body, err := sockRequestRaw(method, endpoint, jsonData, "application/json")
 	if err != nil {
-		b, _ := ioutil.ReadAll(body)
+		if body != nil {
+			b, _ = ioutil.ReadAll(body)
+		}
 		return -1, b, err
 	}
-	var b []byte
-	b, err = readBody(body)
+	if body != nil {
```

Comment:
I think this check isn't needed cause if we are here `sockRequestRaw` hadn't error out and we can assume body isn't nil (the only code paths where body can be nil is from sockRequestRaw erroring out)